.storybook/test-data.js

```diff
@@ -1147,21 +1147,35 @@ const state = {
         v: '0x93',
       },
     ],
-    cachedBalances: {
+    accountsByChainId: {
       1: {
```

Comment:
these should probably be changed to hex chain ID? I think that's what they actually are for accountsByChainId at least

Reply:
agreed. Obviously this state isn't actually being used, I was debating just removing it...

Reply:
fixed here: https://github.com/MetaMask/metamask-extension/pull/22276/commits/62093184c9f8e29a07ff8c7697e0c09982b0007a